db/dynamodb-global-table.yml

```diff
@@ -50,6 +52,10 @@ Metadata:
         default: Primary key field
       PrimaryType:
         default: Primary key type
```

Comment:
Could be changed to "Partition key type"

Reply:
done